modules/ossm-accessing-kiali.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-accessing-kiali_{context}"]
+= Accessing the Kiali console
+
+////
+(how to find the URL to get to the Kiali console)
+Installed Operators > Kiali Operator > Kiali > Kiali Details > Resources > Route > Location = Link
+Networking > Routes> search Kiali route (Location = Link)
+CLI = oc get routes
+////
+
+The installation process creates a route to access the Kiali console.
+
+.Procedure
+
+. Log in to the {Product-title} console.
+
+. Use the perspective switcher to switch to the Administrator perspective.
+
+. Click *Home* -> *Projects*.
+
+. Click the name of your project. For example click `bookinfo`.
+
+. In the Launcher section, click `Kiali`.
+
+. Log in to the Kiali console with the same user name and password that you use to access the {product-title} console.
+
+When you first log in to the Kiali Console, you see the Overview page which displays all the namespaces in your mesh that you have permission to view.
+
+If you are validating the console installation, there may not be any data to display.
```

Comment:
As per ISG:
"Use with caution may   verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead."
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m